src/ComposerRequireChecker/DependencyGuesser/GuessFromComposerAutoloader.php

```diff
@@ -0,0 +1,40 @@
+<?php
+
+namespace ComposerRequireChecker\DependencyGuesser;
+
+
+use Composer\Autoload\ClassLoader;
+
+class GuessFromComposerAutoloader implements GuesserInterface
+{
+
+    /**
+     * @var ClassLoader
+     */
+    private $composerAutoloader;
+
+    private $configVendorDir;
+
+    public function __construct(string $composerJsonPath)
+    {
+        $composerJson = json_decode(file_get_contents($composerJsonPath), true);
```

Comment:
Probably better to require the composer DSL to already be in `array` shape, instead of adding I/O and nullability in here